pkg/agent/util/iptables/iptables.go

```diff
@@ -0,0 +1,96 @@
+package iptables
```

Comment:
should this file be renamed to `ipatbles_linux.go` as part of this PR?

Reply:
iptables is Linux only, so I feel no need to add _linux suffix.

Reply:
I haven't tried in a while, but if there is no `_linux` suffix, won't go try to build this code on Windows, in which case there will be an error since `github.com/coreos/go-iptables/iptables` is Linux-only? Or will go *not* build this code?

Reply:
go won't build this code because this package is not referenced by any code on windows platform. (`util/ipset` and `util/iptables` are only imported by route_linux.go)
I see it's common that if the whole package is OS specific and will only be imported on one OS, no suffix is needed.
https://github.com/kubernetes/kubernetes/tree/master/pkg/proxy/winkernel
https://github.com/kubernetes/kubernetes/tree/master/pkg/util/sysctl
If this file is in `pkg/agent/route/` which will be imported on both platform, it must have linux suffix.

Reply:
Thanks for the info.